lib/loader.js

```diff
@@ -22,6 +22,14 @@ module.exports = function(content, map) {
 	if(sourceMap) {
 		if (map && typeof map !== "string") {
 			map = JSON.stringify(map);
+
+			if (map.sourceRoot) {
+				map.sourceRoot = '';
+			}
+
+			if (map.sources) {
+				map.sources = map.sources.map((source) => source.replace(/\\/g, '/'));
```

Comment:
Neither do I 😛 and I'm really favour make the switch as quickly as possible (node `< 4.0.0` is dead), but if we 'break' things all kind of conservatives will come out of their rat holes  and complain like crazy about it :D 'breaks app IN PRODUCTION' 'USE SEMVER!!!` etc...